app/javascript/dashboard/components/app/UpdateBanner.vue

```diff
@@ -48,26 +41,33 @@ export default {
     },
     shouldShowBanner() {
       return (
+        !this.userDismissedBanner &&
         this.globalConfig.displayManifest &&
-        this.hasAnUpdateAvailable &&
+        this.updateAvailable &&
         !this.isVersionNotificationDismissed(this.latestChatwootVersion) &&
         this.isAdmin
       );
     },
   },
   methods: {
     isVersionNotificationDismissed(version) {
-      return dismissedUpdates.get().includes(version);
+      const dismissedVersions =
+        LocalStorage.get(LOCAL_STORAGE_KEYS.DISMISSED_UPDATES) || [];
+      return dismissedVersions.includes(version);
     },
     dismissUpdateBanner() {
-      let updatedDismissedItems = dismissedUpdates.get();
+      let updatedDismissedItems =
+        LocalStorage.get(LOCAL_STORAGE_KEYS.DISMISSED_UPDATES) || [];
       if (updatedDismissedItems instanceof Array) {
         updatedDismissedItems.push(this.latestChatwootVersion);
       } else {
         updatedDismissedItems = [this.latestChatwootVersion];
       }
-      dismissedUpdates.store(updatedDismissedItems);
-      this.latestChatwootVersion = this.globalConfig.appVersion;
```

Comment:
This line was throwing an error.

<img width="1433" alt="Screenshot 2022-08-05 at 11 39 51 AM" src="https://user-images.githubusercontent.com/2246121/183012773-ae9c1eeb-1e79-43f6-96eb-69643228cb19.png">